build-ol.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
```

Comment:
Not sure we really need to build-ol, build-wl scripts to do the job of default w/ shell/env var override.  
This is probably just preference and doesn't hugely matter.   

From a DRY perspective, we could have all the OL vars set like:

ol.env

```bash
STACK_NAME="Open Liberty"
STACK_SHORT_NAME="openliberty"
...
```

and then we just do:  
```
source ol.env
build.sh
```

We could keep the comment/descriptions too.    I guess I'm just saying there doesn't need to be logic here, just values.

Reply:
Nice idea - simpler.